controllers/imagejob/imagejob_controller.go

```diff
@@ -82,17 +100,67 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.8.3/pkg/reconcile
 func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	_ = log.FromContext(ctx)
-
-	// your logic here
-
-	// create eraser pod in every node like broadcast job does
-	/// get list of nodes and schdules the pods on each of the nodes using pod.spec.nodename
-	// eventually check if pod fits before doing that
+	nodes := &v1.NodeList{}
+	err := r.List(ctx, nodes)
+	if err != nil {
+		return ctrl.Result{}, err
+	}
 
-	//var imageJob eraserv1alpha1.ImageJob
-	//if err := r.Get
+	// map of node names and runtime
+	nodeMap := processNodes(nodes.Items)
+
+	count := 0
```

Comment:
can we use node name as suffix instead of count?

Reply:
there might be a max length for pod name, can you check that

Reply:
looks like the max is 253, so it should be fine to use the node name as a suffix